ports/esp32/partitions-16MiB-ota.csv

```diff
@@ -0,0 +1,10 @@
+# Partition table for MicroPython with OTA support using 16MB flash
+# Notes: the offset of the partition table itself is set in
+# $IDF_PATH/components/partition_table/Kconfig.projbuild.
+# Name,   Type, SubType, Offset,   Size,     Flags
+nvs,      data, nvs,     0x9000,   0x4000,
+otadata,  data, ota,     0xd000,   0x2000,
+phy_init, data, phy,     0xf000,   0x1000,
+ota_0,    app,  ota_0,   0x10000,  0x180000,
+ota_1,    app,  ota_1,   0x190000, 0x180000,
```

Comment:
I'd suggest making these partitions a bit larger.  The firmware size is already close to 1.5MiByte and the IDF does grow over time (we have no control over that).

Reply:
I bumped the OTA partitions to 2 MiB, that should hopefully be good for a while.

Reply:
Second guessing here.  Do you think that's big enough for the next couple of years at least?  Looks like about 0.5 MiB was gained from 1.10 to what will become 1.17.  That's from January 2019 till now, only 2.5 years.